plugin/trino-iceberg/src/test/java/io/trino/plugin/iceberg/TestIcebergSystemTables.java

```diff
@@ -251,23 +266,162 @@ public void testManifestsTable()
     }
 
     @Test
-    public void testFilesTable()
+    public void testFilesTable() throws IOException
```

Comment:
Move `throws ...` to the next line. Please note that `Exception` is acceptable in the case of test methods.